packages/test-runner-webdriver/src/utils.ts

```diff
@@ -1,17 +1,15 @@
-import WebDriver from 'webdriver';
-
-function getPlatform(c: WebDriver.DesiredCapabilities): string | undefined {
-  return c.platformName || c.platform;
+function getPlatform(c: WebdriverIO.Capabilities): string | undefined {
```

Comment:
Where does this file get this type value from? If it works, it works, but confused that there is no import. The main question is if we're doing something special somewhere else, or relying on something that I can't see, then is it visible to a more experienced WebdriverIO user in a way that would prevent their consumption of this from causing them issues?

Reply:
I tried to use this but got error "Cannot use namespace 'Capabilities' as a type".

```
import type { Capabilities } from '@wdio/types';
```

The namespace is defined [here](https://github.com/webdriverio/webdriverio/blob/133dbeb651fab96391bd0af63dbca4d3d17f7aa0/packages/wdio-types/src/Capabilities.ts#L50-L51), seems like it's done for supporting [custom capabilities](https://webdriver.io/docs/capabilities/#extend-custom-capabilities).

Reply:
Does it run fine in other projects this way. I'm fine with whatever works, it was just surprising to see and would be annoying if this caused some undocumented requirements on consumers.